### DISCUSSION TITLE
Product configurator

### DISCUSSION THREAD

**Discussion by MTSPivato** [medusajs/medusa#7093] [Feature Requests]
Hello, I believe it would be very interesting to have a plugin or some integration that allows product customization, something like this: https://scandiweb.com/blog/ecommerce-product-configurator-examples/

If anyone knows of something like this and could recommend it to me, I would be very grateful!

**Comment by dangersvn** [comment]
Hi  @MTSPivato,
Did you have an idea how to implement it?
Thank you!

**Comment by danielgran** [comment]
Also interested, especially with v2.8.x featured
